### DISCUSSION TITLE
Medium article

### DISCUSSION THREAD

**Discussion by ThomasGorisse** [sceneview/sceneform-android#167] [Ideas]
Hi,

After a quick look at "Sceneform" google results, it appears that all articles are speaking about the old sfb plugin way to use Sceneform.

Anyone to create a quick simple article to explain how to use Sceneform Maintained ?

**Comment by Oleur** [comment]
I might have some time in November for that 🙂

**Comment by ThomasGorisse** [comment]
Cool.

And if, for any reason, the Devfest for example 😉, you come to Nantes, we have $5 of sponsoring to buy you a coffee ☕
